### DISCUSSION TITLE
What is the correct way of making reusable controls?

### DISCUSSION THREAD

**Discussion by JaoHundred** [AvaloniaUI/Avalonia#13157] [Q&A] [answered]
Hi, I'm trying to figure out how I should be doing custom reusable controls, so far out of luck. I've already tried reading and re-reading the documentation, looking other built-in controls code but I cant manage to make this to work, if someone could provide how can I manage that I will be grateful. For more context, xamarin forms and maui were really easy to achieve that I only needed dependency property and ContentView.

![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/c535ba52-59de-404b-a674-79b79b3365e6)
![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/bfd18157-20f7-4247-b773-eead0f2baf4d)
![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/cf4f3867-1f50-410b-8481-fadad102ddb4)

**Comment by timunie** [comment]
Check Avalonia.Samples RatingControlSample

**Comment by JaoHundred** [comment, marked as answer]
I was missing `StyleInclude` in App.axaml.

![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/12690516-2119-4c4b-858c-63cc446f4d75)
